RoboJackets/robocup-software
# 50/50

## Description
The ball handler tactic randomly assigns a robot to either pass or shoot (equal probability).

## Steps to test
### Test Case 1
1. Run Keepaway
2. Observe the robots passing and shooting
3. Profit

Expected result: The robots should pass and shoot
